paymentsheet/src/test/java/com/stripe/android/paymentsheet/repositories/CustomerRepositoryTest.kt

```diff
@@ -233,7 +243,8 @@ internal class CustomerRepositoryTest {
                 failsOnceStripeRepository(),
                 { PaymentConfiguration(ApiKeyFixtures.FAKE_PUBLISHABLE_KEY) },
                 Logger.getInstance(false),
-                workContext = testDispatcher
+                workContext = testDispatcher,
+                errorReporter = FakeErrorReporter()
```

Comment:
This seems wrong, we're passing in a new instance of `FakeErrorReporter`, and checking the instance that's a field in the class. (Looks like there's a few instances of this).

Reply:
yep, thanks for catching this! Fixed